build/prbuild.yml

```diff
@@ -43,6 +43,9 @@ jobs:
   - task: PublishBuildArtifacts@1
     displayName: 'Publish Artifact: drop'
 
+  - task: WinAppDriver.winappdriver-pipelines-task.winappdriver-pipelines-task.Windows Application Driver@0
+    displayName: 'Start - WinAppDriver'
```

Comment:
>Start [](start = 18, length = 5)

Can you comment on the logic for straddling the tests with the WinAppDriver?

Reply:
Yes, this guidance comes from this page: https://github.com/Microsoft/WinAppDriver/wiki/WinAppDriver-in-CI-with-Azure-Pipelines